images/builder-cuda-ppc64le/Dockerfile.cuda-11.0

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda-ppc64le:11.0-devel-ubi8
+FROM nvidia/cuda-ppc64le:11.0.3-devel-ubi8
```

Comment:
Does this need to be done for the x86 side as well?

Reply:
apparently yes.  Just added second commit to cover that